101-aci-dynamicsnav/azuredeploy.json

```diff
@@ -99,6 +105,14 @@
                 { 
                   "name": "PublicDnsName",
                   "value": "[variables('publicdnsname')]"
+                },
+                {
+                  "name": "folders",
+                  "value": "c:\\run\\my=https://github.com/NAVDEMO/LetsEncrypt/archive/master.zip\\LetsEncrypt-master"
```

Comment:
this file needs to come from this repo

Reply:
@bmoore-msft Then I can't really test before the merge, right?

Reply:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#deployment-artifacts-nested-templates-scripts) for how to handle artifacts